examples/Vagrantfile

```diff
@@ -53,7 +52,7 @@ Vagrant.configure(2) do |config|
     cloudstack.service_offering_name = "light.S2"
 
     # cloudstack-api listNetworks
-    cloudstack.network_name = "71000001146:10.102.36.0/23"
+    cloudstack.network_name = CHANGEME
```

Comment:
if this is a working example, s/CHANGEME/"CHANGEME"/ to show that it must be a string.